pxtblocks/codecardRenderer.ts

```diff
@@ -100,7 +100,13 @@ export function renderCodeCard(card: pxt.CodeCard, options: CodeCardRenderOption
         }
         if (card.description) {
             const descr = div(ct, 'ui description');
-            const shortenedDescription = card.description.split('.')[0] + '.';
+            const regex = /((?:\.\.\.)|[\!\.\?…])/;
```

Comment:
Nit:
```suggestion
            const regex = /((?:\.{1,3})|[\!\?…])/;
```
However, I don't feel strongly about it. Going with the above suggestion would also mean that we would replace `..` if it was used as punctuation. Not sure if we want that or we want that explicit three periods check.